homeassistant/components/lamarzocco/config_flow.py

```diff
@@ -191,6 +199,45 @@ async def async_step_machine_selection(
             errors=errors,
         )
 
+    async def async_step_bluetooth_selection(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle Bluetooth device selection."""
+
+        assert self.reconfigure_entry
+
+        if user_input is not None:
+            return self.async_update_reload_and_abort(
+                self.reconfigure_entry,
+                data={
+                    **self._config,
+                    CONF_MAC: user_input[CONF_MAC],
+                },
+                reason="reconfigure_successful",
+            )
+
+        bt_options = [
+            SelectOptionDict(
+                value=device_mac,
+                label=device_name,
```

Comment:
As I don't know how the device names look like maybe add also the mac to the label for easier picking by the user in the case it's needed?